main.html

```diff
@@ -189,21 +252,29 @@
 
 > 1.
 
+**2GEN:**(in this mode of solving, you can only do <R,U>, which has the moves R, R', R2, U, U', U2)
```

Comment:
Yup, that's why they weren't showing up in the old markup.  Thanks for seeing that.